java-stack/pom.xml

```diff
@@ -25,13 +25,7 @@
         <dependency>
             <groupId>io.nitric</groupId>
             <artifactId>java-sdk</artifactId>
-            <version>0.10.0-SNAPSHOT</version>
-        </dependency>
-        <dependency>
-            <groupId>io.nitric</groupId>
-            <artifactId>java-sdk-mock</artifactId>
-            <version>0.10.0-SNAPSHOT</version>
-            <scope>test</scope>
+            <version>SNAPSHOT</version>
```

Comment:
Ready for merge once SDK changes are in and auto published (need to update this dependency)